gensim/test/test_keyedvectors.py

```diff
@@ -255,6 +255,15 @@ def test_add_multiple(self):
         for ent, vector in zip(entities, vectors):
             self.assertTrue(np.allclose(kv[ent], vector))
 
+    def test_add_type(self):
+        kv = EuclideanKeyedVectors(2)
+        words, vectors = ["a"], np.array([1., 1.], dtype=np.float32).reshape(1, -1)
+
+        assert kv.vectors.dtype == np.float64  # default dtype of empty KV
```

Comment:
Per comment on `keyedvectors.py`, it'd be better to banish any `float64` arrays from appearing here, rather than checking/adapting to it.

Reply:
Fixed e031fc4